README.md

```diff
@@ -30,6 +30,22 @@ make demo
 The build script will then download data file for Doom automatically. SDL2 based window
 should appear when Doom is loaded and executed.
 
+## Run riscv-arch-test
+
+Once the submodule `riscv-arch-test` is pulled, run all the available compliance test via command:
```

Comment:
Improve the descriptions. You should mention the rationale and general RISC-V Arch Test motivation.